templates/travis/.travis/playbook.yml.j2

```diff
@@ -23,7 +23,6 @@
       secret_key: 'secret'
       databases:
         default:
-          ENGINE: {% raw %}"{{ pulp_db_backend }}"{% endraw %}
```

Comment:
This line needs to set the ENGINE to a string that represents postgresql.

Reply:
Please disregard the previous comment. The value is already set as a default here: https://github.com/pulp/ansible-pulp/blob/master/roles/pulp-database/defaults/main.yml#L9